shunting-yard.h

```diff
@@ -265,12 +265,12 @@ struct parserMap_t {
   rCharMap_t cmap;
 
   // Add reserved word:
-  void add(const std::string& word, const rWordParser_t* parser) {
+  void add(const std::string& word, rWordParser_t* parser) {
     wmap[word] = parser;
   }
 
   // Add reserved character:
-  void add(char c, const rWordParser_t* parser) {
+  void add(char c, rWordParser_t* parser) {
     cmap[c] = parser;
   }
```

Comment:
I am curious about this change, this was causing some warning?

Reply:
Actually, I just tried it, and, indeed, it was causing a linter warning. 

Thanks.